react/features/security/components/security-dialog/native/SecurityDialog.js

```diff
@@ -170,13 +170,39 @@ class SecurityDialog extends PureComponent<Props, State> {
                                 ? 'padding' : 'height'
                         }
                     enabled = { true }>
+                    { this._renderNoSecurityOptions() }
```

Comment:
Let's not do this. Instead, let's hide the button if both individual flags are set or if the umbrella flag is set.

Reply:
Removed it.